packages/bot/src/assets/sass/app.scss

```diff
@@ -1,2 +1,30 @@
 // Fonts
-@import 'deriv-shared/src/styles/google-fonts.scss';
\ No newline at end of file
+@import 'deriv-shared/src/styles/google-fonts.scss';
+
+.btn--icon {
+    display: flex;
+
+    > * {
+        align-self: center;
+    }
+    &-text {
+        color: $COLOR-WHITE;
+        font-size: 1.4em;
+        font-weight: bold;
+        margin-left: 0.5em;
+    }
+}
+
+button:focus {
```

Comment:
@sam-binary we need to use BEM naming convention everywhere. this will change styles for every buttons